### ISSUE TITLE
Version 25+ not supported

### ISSUE THREAD

**Issue by jamesu** [portify/io_scene_dts#27]
Looks like the code doesn't support version 25 and above, used in T3D which might be useful if you're  trying to import or export more recent shapes.

For ref:
- version 25 has secondary uvs, colours, and a slight hack in the generation code to support more indices in each mesh. 
- version 27 (also known as version 26) can store verts directly in an assembled buffer (reducing load times in some cases), but otherwise doesn't have any new data.

Great work on this BTW!

**Comment by qoh** [comment]
Yeah, I'd definitely like to support newer versions as well, that's one of the goals. However, at the moment there's several things I'd like to fix even with the v24 format first.

The reason I originally implemented version 24 first is because that's what [Blockland](http://blockland.us) uses.

Importing different versions should be simple enough. As for exporting I'll likely just have a dropdown in the export options with different versions. Is 24, 25 and 26/27 all it would really need?
